### DISCUSSION TITLE
Help in installing the latest version -v1.5.10

### DISCUSSION THREAD

**Discussion by Banakshi-01** [thoughtspot/cs_tools#174] [Q&A] [answered]
Hi Team , i tried upgrading the CStools version using the self update utility but throws error as "  Failed with exit code: 1

PIP COMMAND BELOW
/cs_tools/.cs_tools/Scripts/python.exe -m pip install --isolated --no-cache-dir --disable-pip-version-check --trusted-host files.pythonhost.org --trusted-host pypi.org
--trusted-host pypi.python.org --trusted-host github.com --trusted-host codeload.github.com cs_tools[cli]  @ https://github.com/thoughtspot/cs_tools/archive/v1.5.10.zip --upgrade --upgrade-strategy eager
........................................

I tried running below  : ( Windows )
py -m pip install https://github.com/thoughtspot/cs_tools/archive/v1.5.10.zip --upgrade --upgrade-strategy eager

This upgraded the tool to 1.5.9 but the packages that was extracted read 1.5.10.

Could someone help guide how to get to 1.5.10

**Comment by boonhapus** [comment, marked as answer]
@Banakshi-01 can you try reinstalling with the following command?

```
powershell -c "IRM https://raw.githubusercontent.com/thoughtspot/cs_tools/master/cs_tools/updater/_bootstrapper.py | python - --reinstall"
```

**Reply by Banakshi-01** [reply]
Thanks . This worked
